py/desitarget/contammask.py

```diff
@@ -10,6 +10,10 @@
     # This can happen during documentation builds.
     #
     def BitMask(*args): return None
+#ADM this is a way to force different behavior during sphinx builds
```

Comment:
Why is `BitMask` defined in both contammask and targetmask?  The *code* should exist in one and only one place, and then the *configuration* of the masks can be in different places.

Reply:
I probably inadvertently duplicated this code when working on the mock target contaminants.  @geordie666 if you could fix this I'd be grateful.